member.php

```diff
@@ -743,6 +743,11 @@
 				my_setcookie("coppauser", 1, -0);
 				$under_thirteen = true;
 			}
+			else
+			{
+				my_setcookie("coppauser", 0, -0);
```

Comment:
I haven't tested this but I think it's incorrect.
The code only check the existence of that specific cookie but not its value:
https://github.com/mybb/mybb/blob/bc8e0c6bcf068b9787bb75f415710660c2371e60/member.php#L394-L400

Reply:
As per @dvz suggestion, the following line :
```PHP
if($mybb->settings['coppa'] != "disabled" && ($mybb->cookies['coppauser'] == 1 || $under_thirteen))
```

Should be :
```PHP
if($mybb->settings['coppa'] != "disabled" && (!empty($mybb->cookies['coppauser']) || $under_thirteen))
```

Finally, after line :
```PHP
my_setcookie("coppadob", "{$mybb->input['bday1']}-{$mybb->input['bday2']}-{$birthday_year}", -1);
```

Add :
```PHP
$under_thirteen = false;
```

This is the simplest solution, avoiding any possible issues like the one suggested by @Stefan-MyBB 

The 2 changes from `isset(...)` to `empty(...)` on lines 394 and 400 seem to be fine.

Reply:
Thank you, should be fixed right now!